pkg/handler/rpms.go

```diff
@@ -167,3 +168,37 @@ func (rh *RpmHandler) listSnapshotRpm(c echo.Context) error {
 
 	return c.JSON(200, setCollectionResponseMetadata(&api.SnapshotRpmCollectionResponse{Data: data}, c, int64(total)))
 }
+
+// detectRpmsPresence godoc
+// @Summary      Detect RPMs presence
+// @ID           detectRpm
+// @Description  This enables users to detect presence of RPMs (Red Hat Package Manager) in a given list of repositories.
+// @Tags         repositories,rpms
+// @Accept       json
+// @Produce      json
+// @Param        body  body   api.DetectRpmsRequest  true  "request body"
+// @Success      200 {object} api.DetectRpmsResponse
+// @Failure      400 {object} ce.ErrorResponse
+// @Failure      401 {object} ce.ErrorResponse
+// @Failure      404 {object} ce.ErrorResponse
+// @Failure      415 {object} ce.ErrorResponse
+// @Failure      500 {object} ce.ErrorResponse
+// @Router       /rpms/presence [post]
+func (rh *RpmHandler) detectRpmsPresence(c echo.Context) error {
+	_, orgId := getAccountIdOrgId(c)
+	dataInput := api.DetectRpmsRequest{}
+	if err := c.Bind(&dataInput); err != nil {
+		return ce.NewErrorResponse(http.StatusBadRequest, "Error binding parameters", err.Error())
+	}
+
+	for i, url := range dataInput.URLs {
+		dataInput.URLs[i] = removeEndSuffix(url, "/")
+	}
```

Comment:
Does this need to be called if CleanupURL() is called in the dao method? This isn't necessarily a suggestion, I don't know the answer :) .

Reply:
ah good catch, i don't think it does! i'll remove it